server/src/main/kotlin/fi/oph/kitu/yki/suoritukset/YkiSuoritusRepository.kt

```diff
@@ -141,43 +148,31 @@ class CustomYkiSuoritusRepositoryImpl : CustomYkiSuoritusRepository {
             } as Iterable<S>
     }
 
-    override fun findAllDistinct(): Iterable<YkiSuoritusEntity> {
+    override fun findAllOrdered(orderBy: String): Iterable<YkiSuoritusEntity> {
+        val findAllQuerySql =
+            """
+            SELECT * FROM
+                (SELECT
+                    $allColumns
```

Comment:
Eikö tässä voisi käyttää * ?
```suggestion
                    *
```